src/routes/about/+page.ts

```diff
@@ -1,64 +0,0 @@
-import type { PageLoad } from './$types';
```

Comment:
it might be good to add a redirect to `/` (I guess will just add issues and get the redesign live)